treeherder/perf/management/commands/import_perf_data.py

```diff
@@ -48,32 +48,31 @@ def _add_series(pc, project_name, signature_hash, signature_props, verbosity, pa
 
     repository = Repository.objects.get(name=project_name)
 
+    defaults = {
+        'test': signature_props.get('test', ''),
+        'suite': signature_props['suite'],
+        'option_collection': option_collection,
+        'platform': platform,
+        'framework': framework,
+        'extra_properties': extra_properties,
+        'has_subtests': signature_props.get('has_subtests', False),
```

Comment:
I hope I understood what you meant, i.e. that we should not set the 'parent_signature' property if there is no parent signature (instead of setting it to 'None'). 

Am I handling 'has_subtests' correctly, or should we also be omitting that instead of setting it to 'False' if it's not in the 'signature_props' dict?